pkgs/by-name/bu/butler/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  pkgs,
+  lib,
+  fetchFromGitHub,
+  Cocoa,
+  ...
+}:
+
+# update instrctions:
+# - Check if butler version bug was fixed https://github.com/itchio/butler/issues/266
+# - if it's fixed, remove patch.
+# - if it was not fixed, follow steps below to regenerate the patch
+# - manually clone butler, change go.mod's version number to 1.18 at least
+# - run `go mod tidy` in the cloned repository.
+# - generate patch with `git diff > go.mod.patch`
+
+pkgs.buildGoModule rec {
+  pname = "butler";
+  version = "15.24.0";
+
+  buildInputs =
+    [
+      pkgs.brotli
+    ]
+    ++ lib.optionals pkgs.stdenv.isDarwin [
+      Cocoa
+    ];
```

Comment:
```suggestion
```

Reply:
Good to know this isn't required, I wasn't able to test it regardless.